Tools/Scripts/webkitdirs.pm

```diff
@@ -3352,12 +3352,13 @@ sub runIOSWebKitApp($)
     die "Not using an iOS SDK."
 }
 
-sub archCommandLineArgumentsForRestrictedEnvironmentVariables()
```

Comment:
This subroutine is exported at the top of the file, please remove the export. I verified that it's not needed any more.

```
       &archCommandLineArgumentsForRestrictedEnvironmentVariables
```

Reply:
Do I need to add the new subroutine to the export, or just removing the old one is enough? 

(Btw, this is the first time I write Perl, so I might ask some silly questions😁.)

Reply:
Please just remove it - it was used in other files before, but not any more.